_posts/2024-03-30-tyc-workshop.md

```diff
@@ -0,0 +1,25 @@
+---
+layout: post
+title: MDAnalysis Workshop with the Thomas Young Centre and CCPBioSim
+---
+
+MDAnalysis, in collaboration with the [Thomas Young Centre](https://thomasyoungcentre.org/), the [JCMaxwell Node of CECAM](https://uk-jcmaxwell.cecam.org/), and [CCPBioSim](https://www.ccpbiosim.ac.uk/), will deliver a hybrid workshop on an **Introduction to Molecular Dynamics Trajectory Analysis using MDAnalysis**. The workshop will take place at the University College London (United Kingdom) and online May 10, 2024 from 09:00 - 17:00 GMT.
```

Comment:
Is there really an article in front of "University College London"?

Reply:
no 😉

Reply:
oops :upside_down: